components/crypto/src/lib.rs

```diff
@@ -578,6 +586,14 @@ mod tests {
         assert_eq!(s, sh);
     }
 
+    #[test]
+    fn to_from_string_public_key() {
+        let (p, _s) = gen_keypair();
```

Comment:
Nit: `_s` could be replaced with `_`. Also, I don't quite like one-char variable names 🙃